.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,67 @@
+# See also https://github.com/EVerest/everest-core/tree/main/.github/workflows
+# TODO: modify to reuse the above workflow to DRY up CI.
+
+name: Build and test libtimer
+on: 
+  pull_request: {}
+  push:
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
+jobs:
+  lint:
+    name: Lint
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+    runs-on: ${{ matrix.os }}
```

Comment:
> Yes, my suggestion was failing, because it had still a typo.
Please use
>
> ```
> runs-on: ${{ inputs.runner || 'ubuntu-22.04' }}
> ```
> or remove the input. Otherwise it is confusing since the input hasn't any effect at the moment

same here